modules/xmpp/JingleSessionPC.js

```diff
@@ -405,7 +412,8 @@ JingleSessionPC.prototype.setLocalDescription = function (sdp, success,
             logger.error('setLocalDescription failed', error);
             if (failure)
                 failure(error);
-            self.room.eventEmitter.emit(XMPPEvents.CONFERENCE_SETUP_FAILED);
+            self.room.eventEmitter.emit(XMPPEvents.CONFERENCE_SETUP_FAILED,
+                error);
```

Comment:
So in this file there are now 2 room.eventEmitter.emit(XMPPEvents.CONFERENCE_SETUP_FAILED with error and 1 without error? Is that by design?